pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -130,25 +137,27 @@ func (p Provisioner) Grant(options *apibkt.BucketOptions) (*bktv1alpha1.ObjectBu
 	}
 
 	// need to quota into -1 for restricting creation of new buckets in rgw
-	_, err = cephObject.SetQuotaUserBucketMax(p.objectContext, p.cephUserName, -1)
+	unlimitedBuckets := -1
```

Comment:
nit: above commands says `-1` represents restricting of creating new buckets, but variable `unlimitedBuckets` is bit confusing here

Reply:
Oh I thought `-1` meant unlimited buckets. Thanks